plugins/modules/swim_workflow_manager.py

```diff
@@ -1861,6 +1877,7 @@ def get_diff_activation(self):
         activation_task_dict = {}
 
         for device_uuid in device_uuid_list:
+            self.log("Stating image activation for multiple devices")
```

Comment:
what image ?? Check the above log messages and make more meaningful

Reply:
updated the code